admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -24,7 +24,7 @@ The LDAP application supports:
 * Support for primary groups in Active Directory
 * Auto-detection of LDAP attributes such as base DN, email, and the LDAP server 
   port number
-* Read-only access to your LDAP (no edit or delete of users on your LDAP) 
+* only read access to your LDAP (no edit or delete of users on your LDAP supported)
```

Comment:
Maybe:
Only - start with capital letter like the other points above it
(edit or delete of users on your LDAP is not supported)